@here/harp-debug-datasource/lib/DebugTileDataSource.ts

```diff
@@ -113,6 +113,8 @@ export class DebugTile extends Tile {
 }
 
 export class DebugTileDataSource extends DataSource {
+    readonly loadable = false;
```

Comment:
This is kinda hacky, it overrides the non-writable value in the base class. It works, but is bad style. Better:
* make "loadable" protected in base class
* set this.loadable in the constructors to false
* please rename "loadable" to "useGeometryLoader"

Reply:
Good point, thanks!